packages/react-composites/src/composites/common/CaptionsBanner.tsx

```diff
@@ -0,0 +1,24 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT license.
+import React from 'react';
+import { _CaptionsBanner } from '@internal/react-components';
+import { _DrawerMenu, _DrawerMenuItemProps, _DrawerSurface } from '@internal/react-components';
+import { Stack } from '@fluentui/react';
+import { usePropsFor } from '../CallComposite/hooks/usePropsFor';
+
+/** @private */
+export const CaptionsBanner = (): JSX.Element => {
+  const captionsBannerProps = usePropsFor(_CaptionsBanner);
+
+  return (
+    <>
+      {captionsBannerProps.captions.length > 0 && (
+        <Stack horizontalAlign="center">
+          <Stack.Item style={{ width: '50%' }}>
```

Comment:
What does width:50% do here? it feels not that future proof to set width in a Stack.item (which is actually a flexbox), let's use something like flex-grow, flex-shrink or properties designed for flexbox